sources/packages/api/src/route-controllers/restriction/restriction.aest.controller.ts

```diff
@@ -252,12 +260,10 @@ export class RestrictionController extends BaseController {
    * @param institutionId
    * @returns Institution Restrictions.
    */
-  @Groups(UserGroups.AESTUser)
-  @AllowAuthorizedParty(AuthorizedParties.aest)
   @Get("/institution/:institutionId")
   async getInstitutionRestrictions(
     @Param("institutionId") institutionId: number,
-  ): Promise<RestrictionSummaryDTO[]> {
+  ): Promise<RestrictionSummaryAPIOutDTO[]> {
```

Comment:
The method `getInstitutionRestrictionsById` will not return `null | undefined` the null check is not needed.